lib/griddler/configuration.rb

```diff
@@ -48,7 +45,9 @@ def reply_delimiter
     end
 
     def email_service
-      @email_service_adapter ||= Griddler.adapter_registry[:default]
+      @email_service_adapter ||=
+        Griddler.adapter_registry[:default] ||
+        raise(Griddler::Errors::EmailServiceAdapterNotFound)
```

Comment:
Use 2 (not 0) spaces for indenting an expression in an assignment spanning multiple lines.